graph/bolt/quadstore.go

```diff
@@ -274,7 +274,7 @@ func (qs *QuadStore) ApplyDeltas(deltas []graph.Delta, ignoreOpts graph.IgnoreOp
 				if err == graph.ErrQuadNotExist && ignoreOpts.IgnoreMissing {
 					continue
 				}
-				return err
+				return &graph.DeltaError{Delta: d, Err: err}
```

Comment:
Does this means that ApplyDeltas should return DeltaError in most cases? There are multiple brunches in this function that doesn't follow this behavior.